hail/python/test/hail/expr/test_ndarrays.py

```diff
@@ -311,6 +316,11 @@ def test_ndarray_ops():
         (ncube1 / nrow_vec, cube1 / row_vec),
         (nrow_vec / ncube1, row_vec / cube1))
 
+    # Missingness tests
```

Comment:
better to create a separate test method with a good name than to add a comment

Reply:
This is consistent with the rest of the `test_ndarray_map2`, which is broken up into sections with comments. I think one test method per IR node is pretty reasonable here. Otherwise I'm going to have separate missingess test method for every one of these

Reply:
heh. We can leave it. I'm not a fan of this single-method, giant-list-of-assertions style of testing. I'd prefer to get all my failures at once instead of one-by-one each time I fix a bug failing a previous assertion.